docs/quickstart.md

```diff
@@ -0,0 +1,139 @@
+Modify and export the following environment variables if you wish to redirect CSO and CSPO to an alternative Git repository
+
+Be aware that GitHub enforces limitations on the number of API requests per unit of time. To overcome this,
+it is recommended to configure a personal access token for authenticated calls. This will significantly increase the rate limit for GitHub API requests.
+
+```bash
+export GIT_PROVIDER_B64=Z2l0aHVi  # github
+export GIT_ORG_NAME_B64=U292ZXJlaWduQ2xvdWRTdGFjaw==  # SovereignCloudStack
+export GIT_REPOSITORY_NAME_B64=Y2x1c3Rlci1zdGFja3M=  # cluster-stacks
+export GIT_ACCESS_TOKEN_B64=<my-github-access-token>
+```
+
+### CSO and CSPO deployment
+
+Install the [envsubst] Go package. It is required to enable the expansion of variables specified in CSPO and CSO manifests.
+
+```bash
+GOBIN=/tmp go install github.com/drone/envsubst/v2/cmd/envsubst@latest
+```
+
+Get the latest CSO release version and apply CSO manifests to the management cluster.
+
+```bash
+# Get the latest CSO release version
+CSO_VERSION=$(curl https://api.github.com/repos/SovereignCloudStack/cluster-stack-operator/releases/latest -s | jq .name -r)
+# Apply CSO manifests
+curl -sSL https://github.com/sovereignCloudStack/cluster-stack-operator/releases/download/${CSO_VERSION}/cso-infrastructure-components.yaml | /tmp/envsubst | kubectl apply -f -
+```
+
+Get the latest CSPO release version and apply CSPO manifests to the management cluster.
+
+```bash
+# Get the latest CSPO release version
+CSPO_VERSION=$(curl https://api.github.com/repos/SovereignCloudStack/cluster-stack-provider-openstack/releases/latest -s | jq .name -r)
+# Apply CSPO manifests
+curl -sSL https://github.com/SovereignCloudStack/cluster-stack-provider-openstack/releases/download/${CSPO_VERSION}/cspo-infrastructure-components.yaml | /tmp/envsubst | kubectl apply -f -
+```
+
+## Create the workload cluster
+
+To transfer the credentials stored in the mentioned secret [above](#create-a-secret-for-openstack-access) to the operator,
+create an `OpenStackClusterStackReleaseTemplate` object and specify this secret in the `identityRef` field.
+The `clouds.yaml` file may contain one or more clouds, so users must specify the desired connection to a specific cloud using the `cloudName` field.
+Refer to the `examples/cspotemplate.yaml` file for more details.
+
+Next, apply this template to the management cluster:
+
+```bash
+kubectl apply -f <path-to-openstack-clusterstack-release-template>
+```
+
+Proceed to apply the `ClusterStack` to the management cluster. For more details, refer to `examples/clusterstack.yaml`:
+
+```bash
+kubectl apply -f <path-to-openstack-clusterstack>
+```
+
+Please be patient and wait for the operator to execute the necessary tasks.
+If your `ClusterStack` object encounters no errors and `openstacknodeimagereleases` is ready, you can deploy a workload cluster.
```

Comment:
IMO this is only partially true. There are other objects openstackclusterstackrelease and clusterstackrelease, which can be ready. ClusterStackRelease needs to be ready and ClusterStack has to report usable versions.

Reply:
fixed: c380e5fa26ad5d4725148dfaf08bed0f842afcc8